### DISCUSSION TITLE
How to differentiate the protein quantification information in each labeling channel

### DISCUSSION THREAD

**Discussion by Jiaxi-UofT** [vdemichev/DiaNN#678] [Q&A]
Hi, I used DIANN to search for multi-labeled data and found that the protein quantification information (PG.Quantity) does not distinguish the proteins in each labeling channel. Did I set it up incorrectly during the search, and is there any way to differentiate the protein quantification information in each labeling channel?

**Comment by vdemichev** [comment]
Yes, when using multiplexing proteins need to be quantified in R (using iq or diann R packages).

Best,
Vadim
